config/locales/en/claims/schools/claims.yml

```
@@ -40,6 +46,13 @@ en:
           window_closed_guidance_link_text: add a mentor
           window_closed_support: Email %{link_to} if you have a query about your claims.
           upcoming_window_guidance: A second claim window will open %{starts_on} and will close %{ends_on} to enable you to submit claims for any remaining mentor training for the %{academic_year_name} academic year.
+          no_claims_for_selected_academic_year: There are no claims for %{school} in the %{academic_year} academic year.
```

```suggestion
          no_claims_for_selected_academic_year: There are no claims for %{school_name} in the %{academic_year} academic year.
```

I was worried that this might output an object, could we rename this to match the input please?